.travis.yml

```diff
@@ -0,0 +1,68 @@
+language: python
+os: linux
+dist: bionic
+
+branches:
+  only:
+    - master
+    - /\d+\.\d+.*/
+  except:
+    - gh-pages
+
+before_install:
+  - pip3 install --upgrade pip
+
+install:
+  - pip3 install -r requirements.txt
+  - pip3 install .
+
+before_script:
+  - pip3 install -r tests/requirements.txt
+
+script:
+  - cd tests
+  - pytest
+
+jobs:
+  include:
+    - name: "Build and test documentation"
+      python: 3.7
+      before_script:
+        - pip3 install -r doc/requirements.txt
+      script:
+        - cd doc
+        - make linkcheck
+        - make html
+
+    - name: "Python 3.5 on Ubuntu 18.04"
+      dist: bionic
+      python: 3.5
+
+    - name: "Python 3.6 on Ubuntu 18.04"
+      python: 3.6
+
+    - name: "Python 3.7 on Ubuntu 18.04 + CodeCov"
+      python: 3.7
+      after_success:
+        - pip3 install codecov
+        - codecov
+
+    - name: "Python 3.7 on macOS (xcode11.2)"
+      os: osx
+      osx_image: xcode11.2
+      language: shell
+
+    - name: "Deploy PyPI package"
+      install: echo "Skipping install stage"
+      script: echo "Skipping script stage"
+      before_script: echo "Skipping before_script stage"
+      deploy:
+        edge: true
+        provider: pypi
+        on:
+          tags: true
+        distributions: sdist
+        skip_existing: false
```

Comment:
why is this false?

Reply:
To overwrite potential existing tags (and keep them in sync with the GitHub tags). Though, a bit of trying out with this tells me this `skip_existing` tag is not functional, because PyPI just bounces back if there is already a file there.

Reply:
I guess its only necessary if its uploaded for many python versions and environments.